.github/workflows/manifold.yml

```diff
@@ -219,7 +219,7 @@ jobs:
     - uses: actions/checkout@v3
       with:
         submodules: recursive
-    - uses: cachix/install-nix-action@v15
+    - uses: cachix/install-nix-action@v22
```

Comment:
I later found that the nix non-deterministic issue is not related to this, but updating should not cause any problem.